src/mpyl/utilities/s3/__init__.py

```diff
@@ -15,17 +15,18 @@
 
 @dataclass
 class S3ClientConfig:
-    service_name = "s3"
-    region_name = "eu-central-1"
-    access_key_id: Optional[str]
-    secret_access_key: Optional[str]
     bucket_name: str
     bucket_root_path: str
+    service_name = "s3"
+    region_name = "eu-central-1"
```

Comment:
Can we move these to the mpyl_config as well?

Reply:
we can! see the latest commit https://github.com/Vandebron/mpyl/pull/196/commits/3fa126548a20f4ae853fade46e2ff7897e969b46. 

- I left the service name hardcoded as this client is only for s3 operations
- I took the liberty of renaming the cloudfront related properties to be more meaningful
- I also updated how the object path is constructed as it wasn't working to well locally